cmd/openshift-install/create.go

```diff
@@ -196,12 +213,22 @@ func runTargetCmd(targets ...asset.WritableAsset) func(cmd *cobra.Command, args
 		cleanup := setupFileHook(rootOpts.dir)
 		defer cleanup()
 
+		gatherer.InitializeInvocationMetrics(gatherer.CreateCommandMetricName[cmd.Name()])
+
 		err := runner(rootOpts.dir)
 		if err != nil {
 			logrus.Fatal(err)
 		}
 		if cmd.Name() != "cluster" {
+			gatherer.SendPrometheusInvocationData(gatherer.CurrentInvocationContext)
 			logrus.Infof(logging.LogCreatedFiles(cmd.Name(), rootOpts.dir, targets))
+		} else {
+			if assetStore, err := assetstore.NewStore(rootOpts.dir); err == nil {
+				if installConfig, err := assetStore.Load(&installconfig.InstallConfig{}); err == nil && installConfig != nil {
+					gatherer.UpdateDurationMetrics(installConfig.(*installconfig.InstallConfig).Config.Platform.Name())
+				}
```

Comment:
In all of the commands,you are adding a loading of the InstallConfig asset... did you try just adding the metrics gather to the install config asset directly? When the asset is loaded/generated update the metric?